draft-ietf-ppm-dap.md

```diff
@@ -1675,27 +1661,19 @@ and Helper then run the aggregate initialization flow to accomplish two tasks:
 
 The Leader and Helper initialization behavior is detailed below.
 
-Implementation note: the Leader will generally want to associate each report
-with a single aggregation job, as otherwise the duplicated reports will
-eventually be discarded as a replay. However, it is likely not appropriate to
-directly use the used-ID storage used for replay protection to determine which
-reports can be added to an aggregation job: certain errors (e.g.
-`report_too_early`) allow the report to be added to another aggregation job in
-the future; but storage into the used-ID storage is permanent.
-
 #### Leader Initialization {#leader-init}
 
 The Leader begins the aggregate initialization by sampling a fresh
 AggregationJobID.
 
 Next, for each report in the candidate set, it checks if the report ID
-corresponds to a report ID it has previously stored for this task. If so, it
+corresponds to a report ID it has previously aggregated for this task. If so, it
 marks the report as invalid and removes it from the candidate set.
 
 Next, the Leader decrypts each of its report shares as described in
 {{input-share-decryption}}, then checks input share validity as described in
-{{input-share-validation}}. If either step invalidates the report, the Leader
-rejects the report and removes it from the set of candidate reports.
+{{input-share-validation}}. If either step fails, the Leader rejects the report
+and removes it from the set of candidate reports.
```

Comment:
brevity/clarity: rejecting the report and removing it from the candidate set are the same thing